mbed-trace/mbed_trace.h

```diff
@@ -69,7 +69,12 @@ extern "C" {
 #define MBED_CONF_MBED_TRACE_ENABLE 0
 #endif
 
-#ifndef MBED_CONF_MBED_TRACE_FEA_IPV6
+#ifndef MBED_CONF_NANOSTACK_LIBSERVICE_PRESENT
+/* if libservice presence is not configured, enable it by default */
+#define MBED_CONF_NANOSTACK_LIBSERVICE_PRESENT 1
```

Comment:
Hmm.. is this actually opposite of what you should be doing?

In Mbed OS, if the libservice is not present, the configuration flag is not defined, so your check is going to define it as one. Should it default to zero?

Reply:
This change is for our test environment. Mbed OS 6.3.x configuration system defines flag MBED_CONF_NANOSTACK_LIBSERVICE_PRESENT and it is not defined in our test env.